tests/e2e/tablespaces_test.go

```diff
@@ -492,21 +496,10 @@ var _ = Describe("Tablespaces tests", Label(tests.LabelSmoke,
 			})
 		})
 
-		It(fmt.Sprintf("can create the cluster by restoring from a volume snapshot backup %v", backupName),
+		It(fmt.Sprintf("can create the cluster by restoring from the backup %v using volume snapshot", backupName),
 			func() {
-				By("fetching the volume snapshots", func() {
-					snapshotList, err := getSnapshots(backupName, clusterName, namespace)
-					Expect(err).ToNot(HaveOccurred())
-					Expect(snapshotList.Items).To(HaveLen(len(backupObject.Status.BackupSnapshotStatus.Elements)))
-
-					envVars := testUtils.EnvVarsForSnapshots{
-						DataSnapshot:             snapshotDataEnv,
-						WalSnapshot:              snapshotWalEnv,
-						TablespaceSnapshotPrefix: snapshotTbsEnv,
-					}
-					err = testUtils.SetSnapshotNameAsEnv(&snapshotList, backupObject, envVars)
-					Expect(err).ToNot(HaveOccurred())
-				})
+				err = os.Setenv("BACKUP_NAME", backupName)
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
don't love this. In the documentation we show recovery from snapshots or from object store prominently.
We completed the restore from backup object functionality for completeness.
Why not keep the code we already had?

Reply:
Hmm, I see that for the PITR test you are using the volumes, so I guess you took advantage to test restore from the backup ojbject. OK, makes sense.